src/components/Message/MessageSimple/MessageContent.tsx

```diff
@@ -188,7 +204,6 @@ const MessageContentWithContext = <
     canDeleteMessage,
     canEditMessage,
     customMessageContent,
-    disabled,
```

Comment:
this was never used?

Reply:
it was used prior when this component was memoized but since that logic moved to Message it doesn't need to be used here anymore